open-craft/edx-theme
# SE-800 Using the added header block, use a different logo for these emails

Uses the header block added by https://github.com/edx/edx-platform/pull/19880 to customize the client's logo in Course Highlights emails.

**JIRA tickets**: SE-800

**Dependencies**: https://github.com/edx/edx-platform/pull/19880

**Screenshots**:

@pkulkark -- check your Inbox for a sample email.

**Sandbox URL**:

* https://hawthorn-cloudera.opencraft.hosting
* https://studio-hawthorn-cloudera.opencraft.hosting/

**Testing instructions**:

See https://github.com/edx/edx-platform/pull/19880 -- the sandbox above has this theme installed, and a [cherry-picked version of the edx-platform PR](https://github.com/open-craft/edx-platform/compare/opencraft-release/hawthorn.1-cloudera...open-craft:jill/course-update-header?expand=1).

**Reviewers**
